### DISCUSSION TITLE
Something to Refer a row in another table

### DISCUSSION THREAD

**Discussion by shaikhashraf709** [Budibase/budibase#1239] [Help]
So have 11 tables in my project which are all related via "Created by, Updated by etc." and there are all three screens(List, Details, New) for all the tables.
when i click on "View" in the greed for particular row (in "List" screen) i can navigate to its "Details" screen and i want to navigate to further "details " screen Created by same user. (like, a user have personal details table, bank details table etc)

I know the "Row Id" is different for a row in different table even those are related to one user so i can't use "Row id".

Is there any way to achieve my purpose?

(I cant use only one table because it has so many fields so i have broken it down to 11 tables and created 11 screens for them)

**Comment by shogunpurple** [comment]
Hi @shaikhashraf709 - thanks for taking the time to raise a discussion. I just tried to do this and noticed that it's not immediately obvious. The best way to do this is via a repeater component. Here's a demo showing you how:

https://www.loom.com/share/98ee97696f054778918a9a2cfc4737c5

Over the coming weeks we are rethinking the data binding experience a bit, which should make it easier to manage repeaters and data bindings. This should help with your use case.
